From: Ben Hutchings <ben@decadent.org.uk>
Subject: crash_dump: Avoid ABI change in 3.12.9
Date: Sat, 01 Feb 2014 18:06:50 +0100

This added #include will probably cause genksyms to change symbol
versions, so hide it.

--- a/include/linux/crash_dump.h
+++ b/include/linux/crash_dump.h
@@ -6,7 +6,9 @@
 #include <linux/proc_fs.h>
 #include <linux/elf.h>
 
+#ifndef __GENKSYMS__
 #include <asm/pgtable.h> /* for pgprot_t */
+#endif
 
 #define ELFCORE_ADDR_MAX	(-1ULL)
 #define ELFCORE_ADDR_ERR	(-2ULL)
